### PR TITLE
Update Portuguese translation for 1.18.0

### PR DESCRIPTION
**Pull request by liddack**

### CODE REVIEW

File: resources/pt/cemu.po [hunk 1]
```diff
@@ -1752,8 +1945,8 @@ msgid "&H264 API"
 msgstr "&H264 API"
 
 #: MainWindow.cpp
-msgid "&NFP API"
-msgstr "API &NFP"
+msgid "&Graphic pack patches"
+msgstr "Correções de &pacotes gráficos"
```
Comment (Fs00) [inline resources/pt/cemu.po:1949]: In this string "patches" refers to the ability of Cemu to apply patches (mods) to games through graphic packs. It seems to me that you're translating "patches" as "corrections", though I may be wrong.

**Comment by Fs00** [comment]
Thanks! I have only a small doubt on a single string.

**Comment by liddack** [comment]
You're not wrong -- `correções` (corrections) is a commonly used translation for `patches`. But now that you mentioned it, I see `correções` is not a good fit in this context, as `patches` has a broader meaning - they might not be (only) "corrections". I took a look at the [Microsoft Terminology database](https://www.microsoft.com/en-us/language/Search?&searchTerm=patch&langID=594&Source=true&productid=0) and saw they don't translate this term, so I'm now following their lead.

Thanks for bringing it to my attention 😁

**Comment by Fs00** [comment]
Thanks! Also I didn't know the existence of that terminology database, it'll come in handy from now on 😃.

**Pull request merged**